tests/assembly/riscv64gc-zbb.rs

```diff
@@ -0,0 +1,23 @@
+//@ assembly-output: emit-asm
+//@ compile-flags: --target riscv64gc-unknown-linux-gnu
+//@ compile-flags: -C target-feature=+zbb
+//@ compile-flags: -C opt-level=3
```

Comment:
This won't be good enough, it needs to have compiled the parts of libcore we use here and that isn't guaranteed on a given test host.

Reply:
I have added `only-riscv64` header for it, it should be fine now.